docs/getting-started-guides/rkt/README.md

```diff
@@ -47,8 +47,73 @@ We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the expe
 
 - Note that for rkt version later than v0.7.0, `metadata service` is not required for running pods in private networks. So now rkt pods will not register the metadata service be default.
 
-- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5), [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
-  is required to be running on the machine.
+- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5),
+the [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
+must be running on the node.
+
+### Network Setup
+
+rkt uses the [Container Network Interface (CNI)](https://github.com/appc/cni)
+to manage container networking. By default, all pods attempt to join a network
+called `rkt.kubernetes.io`, which is currently defined [in `rkt.go`]
+(https://github.com/kubernetes/kubernetes/blob/v1.2.0-alpha.6/pkg/kubelet/rkt/rkt.go#L91).
+In order for pods to get correct IP addresses, the CNI config file must be
+edited to add this `rkt.kubernetes.io` network:
+
+#### Using flannel
+
+[flannel](https://github.com/coreos/flannel) can be configured with a
+CNI config like:
+
+```console
+$ cat <<EOF >/etc/rkt/net.d/k8s_cluster.conf
+{
+    "name": "rkt.kubernetes.io",
+    "type": "flannel"
+}
+EOF
+```
+
+While `k8s_cluster.conf` is a rather arbitrary name for the config file itself,
+and can be adjusted to suit local conventions, the keys and values should be exactly
+as shown above. `name` must be `rkt.kubernetes.io` and `type` should be `flannel`.
+More details about the flannel CNI plugin can be found
+[in the CNI documentation](https://github.com/appc/cni/blob/master/Documentation/flannel.md).
+
+#### On GCE
+
+Each VM on GCE can have up to 256 IP addresses routed to it, so flannel isn't called
+for in most smaller Kubernetes clusters on GCE. This makes the necessary CNI config
+file a bit more verbose:
```

Comment:
Our thinking was that GCE was a handy example of a fairly easy but flexible alternative configuration without flannel. Only the routable ips part is truly specific to the k8s [networking design](https://github.com/kubernetes/kubernetes/blob/master/docs/design/networking.md#implementation) on GCE and that's what we were trying to fit in with.

As mentioned a couple of paragraphs down, the life expectancy of this example is fairly short, but we could leave out the alternate example if you don't think it's useful.